scripts/zkai_attention_kv_stwo_native_d16_bounded_softmax_table_proof_input.py

```
@@ -0,0 +1,688 @@
+                "value": list(candidate["value"]),
+                "products": list(products),
+                "weighted_value": list(weighted_value),
+                "weighted_numerator": list(numerators),
+                "attention_output": list(output),
+                "output_remainder": list(remainders),
+            })
+        current = candidates
+    return rows, current, outputs
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (18 > 12)

(PLR0912)